fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/loanschedule/domain/LoanApplicationTerms.java

```diff
@@ -217,6 +217,8 @@ public final class LoanApplicationTerms {
     private boolean isScheduleExtensionForDownPaymentDisabled;
     private Money disbursedPrincipal;
 
+    private boolean enableInstallmentLevelDlnqcy;
+
```

Comment:
@josehernandezfintecheandomx, not sure if this configuration needs to be passed on for LoanApplicationTerms.
Also please check if Loan needs to inherit this configuration. If yes, changes needs to be implemented for the same.

Reply:
Done! It has been updated and now It's on Loan entity level inherit from the Loan Product at request time

<img width="1365" alt="Screenshot 2023-10-16 at 6 16 10 p m" src="https://github.com/apache/fineract/assets/44206706/5dfbdf5f-7a8b-44d6-8427-a64a89dcf60a">